src/wallet/wallet2.cpp

```diff
@@ -3899,6 +3905,9 @@ bool wallet2::load_keys(const std::string& keys_file_name, const epee::wipeable_
     GET_FIELD_FROM_JSON_RETURN_ON_ERROR(json, encrypted_secret_keys, uint32_t, Uint, false, false);
     encrypted_secret_keys = field_encrypted_secret_keys;
 
+    GET_FIELD_FROM_JSON_RETURN_ON_ERROR(json, export_format, ExportFormat, Int, false, Hex);
```

Comment:
The default should be raw or you're going to switch existing wallets on load.

Reply:
Good catch, thanks.